src/test/java/org/folio/dew/service/ModuleTenantServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package org.folio.dew.service;
+
+import static org.folio.dew.utils.Constants.EUREKA_PLATFORM;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.folio.dew.client.EurekaProxyTenantsClient;
+import org.folio.dew.client.OkapiClient;
+import org.folio.dew.domain.bean.ModuleForTenant;
+import org.folio.spring.FolioExecutionContext;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.net.URI;
+import java.util.List;
+
+@ExtendWith(MockitoExtension.class)
+class ModuleTenantServiceTest {
+
+  @Mock
+  private FolioExecutionContext folioExecutionContext;
+  @Mock
+  private OkapiClient okapiClient;
+  @Mock
+  private EurekaProxyTenantsClient eurekaProxyTenantsClient;
+
+  @InjectMocks
+  private ModuleTenantService moduleTenantService;
+
+  @Test
+  void getModuleIdForOkapiTest() {
+    var tenantId = "diku";
+    var moduleId = "moduleId";
```

Comment:
Is it possible to use not fake moduleId like in test getModuleIdForEurekaTest() - "mod-users-19.4.1-SNAPSHOT.322" instead of "moduleId"?

Reply:
Updated module id like real exists